src/fixtures/telemetry_collectors/nested_collector.ts

```diff
@@ -19,7 +19,7 @@ interface Usage {
 }
 
 export class NestedInside {
-  collector?: UsageCollector<Usage>;
+  collector?: Collector<Usage>;
```

Comment:
Using `Collector` as a general interface. Not exporting `UsageCollector` anymore. This will help in the future when/if we want to merge both entities back into one.